infra/src/main/java/com/redhat/service/smartevents/infra/models/dto/UpdateManagedResourceStatusDTO.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.service.smartevents.infra.models.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class UpdateManagedResourceStatusDTO {
```

Comment:
This will affect my PR https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/968 but this is a good improvement on what we do at the moment so I will rebase and refactor accordingly after this PR is merged (which is likely to happen way before mine!)

Reply:
Yeah I had this refactoring in mind since a bit... It was no sense anymore to return the entire BridgeDTO/ProcessorDTO back to the manager